decompressed/gui_file/usr/share/transformer/scripts/upgradegui

```diff
@@ -156,7 +157,7 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 	
 	# Download new GUI to /tmp
 	if ! $curl $URL_BASE/$FILE_NAME --output $WORKING_DIR/$FILE_NAME; then
-	logger_command "ERROR: can't find new GUI file" >&2
+	logger_command "ERROR: Can't find new GUI file" >&2
```

Comment:
Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'

Reply:
I prefer to leave in lowercase "can't"